spec/Section 3 -- Type System.md

```diff
@@ -2227,3 +2227,160 @@ to the relevant IETF specification.
 ```graphql example
 scalar UUID @specifiedBy(url: "https://tools.ietf.org/html/rfc4122")
 ```
+
+## Schema Coordinates
+
+SchemaCoordinate :
+
+- TypeCoordinate
+- MemberCoordinate
+- ArgumentCoordinate
+- DirectiveCoordinate
+- DirectiveArgumentCoordinate
+
+TypeCoordinate : Name
+
+MemberCoordinate : Name . Name
+
+ArgumentCoordinate : Name . Name ( Name : )
+
+DirectiveCoordinate : @ Name
+
+DirectiveArgumentCoordinate : @ Name ( Name : )
+
+:: A _schema coordinate_ is a human readable string that uniquely identifies a
+_schema element_ within a GraphQL Schema.
+
+:: A _schema element_ can be a named type, a field, an input field, an enum
+value, a field argument, a directive, or a directive argument defined within a
+schema (including built-in types and directives).
+
+Note: Meta-fields are not defined within a schema, and thus are not _schema
+element_. By extension, an introspection type is not a _schema element_.
+
+:: The _containing element_ of a _schema element_ is the schema element with one
+fewer {Name} token that syntactically contains it. Specifically:
+
+- The containing element of an {ArgumentCoordinate} is a {MemberCoordinate}.
+- The containing element of a {MemberCoordinate} is a {TypeCoordinate}.
+- The containing element of a {DirectiveArgumentCoordinate} is a
+  {DirectiveCoordinate}.
+- {TypeCoordinate} and {DirectiveCoordinate} have no containing element.
+
+A _schema coordinate_ is always unique. Each _schema element_ can be referenced
+by exactly one possible schema coordinate.
+
+A _schema coordinate_ may refer to either a defined or built-in _schema
+element_. For example, `String` and `@deprecated(reason:)` are both valid schema
+coordinates which refer to built-in schema elements.
+
+Note: Union members are not valid _schema coordinate_ as they reference existing
+types in the schema. This preserves the uniqueness property of a _schema
+coordinate_ as stated above.
+
+Note: A {SchemaCoordinate} is not a definition within a GraphQL {Document}, but
+a separate standalone grammar, intended to be used by tools to reference types,
+fields, and other _schema element_. Examples include: references within
+documentation to refer to types and fields in a schema, a lookup key that can be
+used in logging tools to track how often particular fields are queried in
+production.
```

Comment:
How about this, @magicmark?

```suggestion
production.

**Lexical Analysis & Syntactic Parse of a Schema Coordinate**

The source text of a _schema coordinate_ is processed in the same way as that of
a GraphQL document, as laid out in the [Language](#sec-Language) section, with
the exception that {Ignored} tokens are disallowed. A schema coordinate must
therefore not contain whitespace, line terminators, comments, commas, or a _Byte
Order Mark_. This constraint ensures that schema coordinates are both
unambiguous and unique.
```

Reply:
Added a "with the exception of _schema coordinate_" here to avoid a contradiction

https://github.com/graphql/graphql-spec/pull/794/commits/f4eec91893d26a65d75bdeba73a6a815190ab710